app/helpers/application_helper/toolbar/physical_server_center.rb

```diff
@@ -1,4 +1,32 @@
 class ApplicationHelper::Toolbar::PhysicalServerCenter < ApplicationHelper::Toolbar::Basic
+  button_group('physical_server_vmdb', [
+    select(
+      :physical_server_vmdb_choice,
+      'fa fa-cog fa-lg',
+      t = N_('Configuration'),
+      t,
+      :items => [
+        button(
+          :physical_server_refresh,
+          'fa fa-refresh fa-lg',
+          N_('Refresh relationships and power states for all items related to the selected Physical Servers'),
+          N_('Refresh Relationships and Power States'),
+          :url_parms => "main_div",
+          :confirm   => N_("Refresh relationships and power states for all items related to the selected Physical Servers?"),
+          :enabled   => false,
```

Comment:
When in the Summary screen, you do not need -
```rb
:enabled   => false,
:onwhen    => "1+"
```
They are applicable in the list view only.
If you add the above entries for the Menu in Summary screen, the `Refresh` and `Remove` buttons would never
be enabled since `onwhen => "1+"` would never be true.